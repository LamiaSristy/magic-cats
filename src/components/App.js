/*eslint-disable*/
import '../stylesheets/App.scss';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import CatList from '../containers/CatList';
import Cat from '../containers/Cat';
import AboutUs from './AboutUs';

const App = () => (
  <div>
    {/* <CatList /> */}
    <BrowserRouter>
        {/* <ScrollToTop /> */}
        <Switch>
          <Route exact path="/">
            <CatList />
          </Route>
          <Route exact path="/cat/:id">
            <Cat />
          </Route>
          <Route exact path="/AboutUs">
            <AboutUs />
          </Route>
          {/* <Route path="/404">
            <FourOFour />
          </Route> */}
        </Switch>
      </BrowserRouter>
  </div>
);
export default App;
