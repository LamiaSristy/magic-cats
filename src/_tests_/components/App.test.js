import React from 'react';
import { shallow } from 'enzyme';
import App from '../../components/App';

const setup = () => shallow(<App />);

describe('App component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });

  it('Should render', () => {
    const element = component.find('.app');
    expect(element.length).toEqual(1);
  });
});
