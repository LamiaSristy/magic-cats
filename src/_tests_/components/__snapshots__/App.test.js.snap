// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders the app in the page 1`] = `ShallowWrapper {}`;
